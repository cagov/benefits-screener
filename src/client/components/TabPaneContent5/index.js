import { Trans, useTranslation } from "react-i18next";
import React from "react";

function TabPaneContent5() {
  const { t } = useTranslation();

  return (
    <div>
      <h3>
        <Trans t={t} i18nKey="tab5-header1" />
      </h3>
      <p>
        <Trans t={t} i18nKey="tab5-p1">
          <a href={t("links.edd-coronavirus-faqs")}>
            Coronavirus (COVID-19) FAQs
          </a>
        </Trans>
      </p>
      <p>
        <Trans t={t} i18nKey="tab5-p2">
          More details about what you need to do after you file your claim:{" "}
          <a href={t("links.edd-coronavirus-claims")}>
            COVID-19 Unemployment Insurance Claims
          </a>
        </Trans>
      </p>
      <h3>
        <Trans t={t} i18nKey="tab5-header2" />
      </h3>
      <p>
        <Trans t={t} i18nKey="tab5-p3" />
      </p>
      <h4>
        <Trans t={t} i18nKey="tab5-header3" />
      </h4>
      <p>
        <Trans t={t} i18nKey="tab5-p4">
          <strong>This expands who can apply for UI, due to COVID-19.</strong>{" "}
          This includes business owners, independent contractors, self-employed
          workers, freelancers, gig workers and people with limited work
          history. It is also available to people who have exhausted their
          regular UI claims, are serving penalty weeks on their claim, or whose
          claim may be disqualified. PUA supports claims between February 2 and
          December, 2020. <a href={t("links.edd-pua")}>Read more about PUA</a>.
        </Trans>
      </p>
      <h4>
        <Trans t={t} i18nKey="tab5-header4" />
      </h4>
      <p>
        <Trans t={t} i18nKey="tab5-p5" />
      </p>
      <h4>
        <Trans t={t} i18nKey="tab5-header5" />
      </h4>
      <p>
        <Trans t={t} i18nKey="tab5-p6" />
      </p>
      <div className="highlight-blockquote">
        For more information, visit{" "}
        <a href="http://https://edd.ca.gov/about_edd/coronavirus-2019/cares-act.htm">
          CARES Act Provisions
        </a>
        .
      </div>
    </div>
  );
}

export default TabPaneContent5;
