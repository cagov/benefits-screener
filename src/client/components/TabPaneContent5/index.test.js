import React from "react";
import TabPaneContent5 from "./index";
import { shallow } from "enzyme";

describe("<TabPaneContent5 />", () => {
  it("renders content pane", async () => {
    const wrapper = shallow(<TabPaneContent5 />);

    expect(wrapper).toMatchSnapshot();
  });
});
