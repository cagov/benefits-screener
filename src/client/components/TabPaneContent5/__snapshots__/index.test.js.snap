// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabPaneContent5 /> renders the TabPaneContent component 1`] = `
<TabPaneContent5>
  <div>
    <h3>
      <Trans
        i18nKey="tab5-header1"
        t={[Function]}
      >
        Helpful links
      </Trans>
    </h3>
    <p>
      <Trans
        i18nKey="tab5-p1"
        t={[Function]}
      >
        <a
          href="https://edd.ca.gov/about_edd/coronavirus-2019/faqs.htm"
          key="0"
        >
          Coronavirus (COVID-19) FAQs
        </a>
      </Trans>
    </p>
    <p>
      <Trans
        i18nKey="tab5-p2"
        t={[Function]}
      >
        More details about what you need to do after you file your claim: 
        <a
          href="https://edd.ca.gov/about_edd/coronavirus-2019/unemployment-claims.htm"
          key="1"
        >
          COVID-19 Unemployment Insurance Claims
        </a>
      </Trans>
    </p>
    <h3>
      <Trans
        i18nKey="tab5-header2"
        t={[Function]}
      >
        New unemployment programs from the CARES Act
      </Trans>
    </h3>
    <p>
      <Trans
        i18nKey="tab5-p3"
        t={[Function]}
      >
        These programs expand Unemployment Insurance (UI) benefits by increasing the number of people who are eligible, increasing benefit amounts, and extending the length of the program.
      </Trans>
    </p>
    <h4>
      <Trans
        i18nKey="tab5-header3"
        t={[Function]}
      >
        Pandemic Unemployment Assistance (PUA)
      </Trans>
    </h4>
    <p>
      <Trans
        i18nKey="tab5-p4"
        t={[Function]}
      >
        <strong
          key="strong-0"
        >
          This expands who can apply for UI, due to COVID-19.
        </strong>
         This includes business owners, independent contractors, self-employed workers, freelancers, gig workers and people with limited work history. It is also available to people who have exhausted their regular UI claims, are serving penalty weeks on their claim, or whose claim may be disqualified. PUA supports claims between February 2 and December, 2020. 
        .
        .
      </Trans>
    </p>
    <h4>
      <Trans
        i18nKey="tab5-header4"
        t={[Function]}
      >
        Federal Pandemic Unemployment Compensation (FPUC) or Pandemic Additional Compensation (PAC)
      </Trans>
    </h4>
    <p>
      <Trans
        i18nKey="tab5-p5"
        t={[Function]}
      >
        <strong
          key="strong-0"
        >
          This provides an additional $600 per week
        </strong>
         to people receiving UI benefits. It is retroactive for claims between March 29 and July 25, 2020. This $600 benefit is issued every two weeks. You will receive it at the same time as your UI or PUA benefits.
      </Trans>
    </p>
    <h4>
      <Trans
        i18nKey="tab5-header5"
        t={[Function]}
      >
        Pandemic Emergency Unemployment Compensation (PEUC)
      </Trans>
    </h4>
    <p>
      <Trans
        i18nKey="tab5-p6"
        t={[Function]}
      >
        <strong
          key="strong-0"
        >
          This provides an additional 13 weeks of benefits
        </strong>
         for people receiving UI benefits between March 29 and December 26, 2020. California provides up to 26 weeks of UI benefits depending on the amount of your earnings during the four-quarter period on which your claim is based. The CARES Act extends benefits to a total of 39 weeks.
      </Trans>
    </p>
    <div
      className="highlight-blockquote"
    >
      <Trans
        i18nKey="tab5-blockquote1"
        t={[Function]}
      >
        <strong
          key="strong-0"
        >
          Note:
        </strong>
         We are still working to implement the PEUC program. Please continue to monitor our website for instructions on next steps for the PEUC program.
      </Trans>
    </div>
  </div>
</TabPaneContent5>
`;
